src/term/mod.rs

```diff
@@ -1109,18 +1187,31 @@ impl ansi::Handler for Term {
         debug_println!("unset mode::APP_KEYPAD");
         self.mode.remove(mode::APP_KEYPAD);
     }
+
+    #[inline]
+    fn designate_charset(&mut self, index: CharsetIndex, charset: StandardCharset) {
```

Comment:
Thought about the name a bit, and something like `configure_charset` seems much more intuitive. What do you think?